packages/ccache.rb

```diff
@@ -3,41 +3,46 @@
 class Ccache < Package
   description 'Compiler cache that speeds up recompilation by caching previous compilations'
   homepage 'https://ccache.samba.org/'
-  @_ver = '4.6.3'
+  @_ver = '4.7.4'
   version @_ver
   license 'GPL-3 and LGPL-3'
   compatibility 'all'
   source_url "https://github.com/ccache/ccache/releases/download/v#{@_ver}/ccache-#{@_ver}.tar.xz"
-  source_sha256 '1e3a251bb112632553b8255a78661fe526c3a16598496d51128c32b218fd8b22'
+  source_sha256 'df0c64d15d3efaf0b4f6837dd6b1467e40eeaaa807db25ce79c3a08a46a84e36'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ccache/4.6.3_armv7l/ccache-4.6.3-chromeos-armv7l.tar.zst',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ccache/4.6.3_armv7l/ccache-4.6.3-chromeos-armv7l.tar.zst',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ccache/4.6.3_i686/ccache-4.6.3-chromeos-i686.tar.zst',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ccache/4.6.3_x86_64/ccache-4.6.3-chromeos-x86_64.tar.zst'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ccache/4.7.4_armv7l/ccache-4.7.4-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ccache/4.7.4_armv7l/ccache-4.7.4-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ccache/4.7.4_i686/ccache-4.7.4-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ccache/4.7.4_x86_64/ccache-4.7.4-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '81425bb2430cf6c88dd4b5a7cc391768ee340d388c290c66bad4e6bb8cdee6d9',
-     armv7l: '81425bb2430cf6c88dd4b5a7cc391768ee340d388c290c66bad4e6bb8cdee6d9',
-       i686: '7d80c4ca1cc5210219e55819b5a8c505c1b6c2ec1868b0c7f844c0ca158870ac',
-     x86_64: 'c1db0bb70b6af2544d4026200a93fdf40080b007a6324d75c06455aad2efaef8'
+    aarch64: '31a4788007867b6a3a102d3da3845b2876870742a8efb6ce8a3a283c69726f95',
+     armv7l: '31a4788007867b6a3a102d3da3845b2876870742a8efb6ce8a3a283c69726f95',
+       i686: 'cb6d8a88eb093a3e6db8b51a5b31d1d69e4a851343d4f3a0b3496d88ff67974b',
+     x86_64: '8577d5280099b3e43e1cbbbac90aa63594191b2a9a5533a432ae5b1edbded944'
   })
 
   depends_on 'xdg_base'
-  depends_on 'asciidoc' => :build
+  depends_on 'ruby_asciidoctor' => :build
 
   def self.build
     Dir.mkdir 'build'
     Dir.chdir 'build' do
-      system "env #{CREW_ENV_OPTIONS} \
-      cmake -G Ninja \
+      system "cmake -G Ninja \
       #{CREW_CMAKE_OPTIONS} \
       -DCMAKE_INSTALL_SYSCONFDIR=#{CREW_PREFIX}/etc \
       -DZSTD_FROM_INTERNET=ON \
       -DHIREDIS_FROM_INTERNET=ON \
       .."
       system 'mold -run ninja'
     end
+    @ccacheenv = <<~CCACHEEOF
+      # ccache configuration
+      if ! [[ $PATH == *"ccache/bin"* ]]; then
```

Comment:
Use `!=`?

Reply:
That might work but this is a regex evaluation.

Reply:
This is globbing... (regex uses `~=` and `!=` as operator)

Reply:
Right, I just tested with `!=` and it works.  Rebuilding binaries... :sleeping:

Reply:
Does using `ccache` make the rebuild go faster? 🤣

Reply:
You asked too late.  Already did the rebuilds.  How do you use this anyway?

Reply:
You add the ccache dir to the path first, and then ccache intercepts calls to the compiler and maintains/checks a cache of compiled code for every compile request.